src/components/Mission.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- Please remove all disabled linters, remember linters help us to ensure that we write code that is clean and easy to maintain, please fix this 🙏

Reply:
Hi @mckent05, That file was unnecessary, just removed it